packages/odyssey-storybook/src/components/odyssey-mui/DataTable/DataTable.mdx

```diff
@@ -418,6 +422,64 @@ Action buttons and menu items can be added to each row. If rows are selectable,
 which will be applied to all selected rows. (For example, the bulk actions menu starts out disabled, but if the user selects 3 rows,
 the actions menu will be enabled and any actions performed would be applied to all 3 selected rows.)
 
+## Custom filters
+
+<Canvas>
+  <Story of={DataTableStories.CustomFilters} />
+</Canvas>
+
+Within `<DataTable>`, you can now pass a custom `filters` prop. _This will override the default filters, which are based off the table columns_.
```

Comment:
Remember to alter this documentation if we change the implementation to a separate boolean prop.